samples/sensor/sgp40_sht4x/boards/blackpill_f411ce.overlay

```diff
@@ -0,0 +1,27 @@
+/*
+ * Copyright (c) 2021 Leonard Pollak
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+&i2c1 {
+	status = "okay";
+	compatible = "st,stm32-i2c-v1";
+
+	clock-frequency = <I2C_BITRATE_FAST>;
+	label = "I2C_0";
```

Comment:
label should already be provided by soc DT

Reply:
removed